lisa/features/serial_console.py

```diff
@@ -58,6 +63,26 @@ def invalidate_cache(self) -> None:
         )
         self._cached_console_log = None
 
+    def get_matched_str(self, pattern: Pattern[str]) -> str:
```

Comment:
Same name as the imported one, so confusing... Also, amazed that our make check does not complain about this empty line here.

Reply:
They perform the same functionality. The called one is for any string, this is in the SerialConsole class, only for serial log.
Remove the empty line to keep consistence.